### ISSUE TITLE
Clicking items shouldn't result in browser nav

### ISSUE THREAD

**Issue by calebtote** [atoddj/squirtle-issues#15]
Clicking different headings shouldn't update browser navigation, ultimately resulting in the user having to press "Back" multiple times to exit the site.

**Comment by atoddj** [comment]
I disagree. If a user hits back expecting to go to a page that he or she
was looking at, it would make them start over if the app doesn't create
something for their history. I think it's more expected to have the nav
updated

**Comment by atoddj** [comment]
Spoke with Caleb and clarified the issue happening. 

When a user navigates in mobile to different pages, the expanded navigation menu should collapse.

**Comment by atoddj** [comment]
http://stackoverflow.com/questions/14741988/twitter-bootstrap-navbar-with-angularjs-collapse-not-functioning?rq=1

I should be able to fix it with this information
